src/pages/Improbability/Improbability.container.tsx

```diff
@@ -0,0 +1,8 @@
+import { ImprobabilityComponent } from './Improbability.component';
+import { mockCoins } from './Improbability.mock';
+
+export const ImprobabilityContainer = () => {
+  const coins = mockCoins;
+  const state = 'idle';
+  return <ImprobabilityComponent coins={coins} state={state} />;
```

Comment:
Imo, `ImprobabilityComponent` should get the `coins` object as `ImprobabilityRowData`. Then there's no need of type assertion when passing to `<DataTable>`, which is always good to avoid, if possible. And the `ImprobabilityComponent` won't be using the addresses anyway